src/wallet/rpcwallet.cpp

```diff
@@ -2117,6 +2117,77 @@ static UniValue listaccounts(const JSONRPCRequest& request)
     return ret;
 }
 
+static UniValue listsincetx(const JSONRPCRequest& request)
+{
+    CWallet* const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
+        throw std::runtime_error(
+            "listsincetx \"txid\" (polltime)\n"
```

Comment:
Could have a `limit` argument? Otherwise it would dump the whole wallet?

Reply:
I thought of it,... but would like to keep it simple for a first implementation (limit could follow next)